File: frontend/lib/features/calendar/domain/entities/schedule.dart
class Schedule {
  final int id;
  final int dayOfWeek;
  final String startTime;
  final String endTime;
  final int duration;
  final String clinicName;

  const Schedule({
    required this.id,
    required this.dayOfWeek,
    required this.startTime,
    required this.endTime,
    required this.duration,
    required this.clinicName,
  });
}

File: backend/laravel/app/Models/Schedule.php
<?php

namespace App\Models;

class Schedule extends BaseModel
{
    protected $table = 'schedules';

    protected $fillable = [
        'id_doctor',
        'id_clinic',
        'day_of_week',
        'start_time',
        'end_time',
        'is_active',
        'duration',
    ];

    protected $casts = [
        'is_active' => 'boolean',
    ];
}
